### ISSUE TITLE
Runner installer doesn't exit gracefully after successful installation. 

### ISSUE THREAD

**Issue by todaywasawesome** [codefresh-io/cli#510]
```
------------------------------
Executing command: echo hello codefresh runner!
hello codefresh runner!

Runner Status:
NAME                                                 RUNTIMES                                             STATUS   REPORTED                
docker-desktop_allinone                              docker-desktop/allinone                              N/A      N/A                     
katacoda                                             kubernetes-admin@kubernetes/allinone                 N/A      N/A                     
gke_codefresh-sa_us-central1-c_dan-cluster_codefresh gke_codefresh-sa_us-central1-c_dan-cluster/codefresh All good 2020-06-18T16:30:00.071Z

Go to https://g.codefresh.io/kubernetes/monitor/services to view your cluster in codefresh dashbaord

Documenation link: https://codefresh.io/docs/docs/enterprise/codefresh-runner/#codefresh-runner-preview-release
If you had any issues with the installation please report them at: https://github.com/codefresh-io/cli/issues/new



```

Everything ran properly and it appears successful but now it's hanging after successful installation.

**Comment by roikramer120** [comment]
Hey @todaywasawesome, from the logs you provided you it appears your installation reached a point where it is supposed to report installation success to the Codefresh platform. What probably happened is that reporting the success status was hanging, maybe due to slow internet connection or maybe due to load on the Codefresh platform. If you would have waited a bit longer the request should have timed out.

**Comment by todaywasawesome** [comment]
@roi-codefresh I let it sit for over an hour, so I'm not sure there's a timeout that catches.

**Comment by todaywasawesome** [comment]
@roi-codefresh tried it again and left it for 40 minutes without successful exit. 

![Screen Shot 2020-06-30 at 12 12 46 PM](https://user-images.githubusercontent.com/3849900/86161843-1dd75480-bacb-11ea-8824-063689a6fea3.png)

**Comment by oren-codefresh** [comment]
@todaywasawesome did you update to the latest version?

**Comment by oren-codefresh** [comment]
It's solved on 0.69.6
